slides/slides.tex

```diff
@@ -124,500 +124,73 @@
-	\end{align*}
-	$\varphi$ is the toroidal angle and 
-	$F_0$ is a constant. 
-
 	\begin{alertblock}{}
-		If $\nabla \chi \approx \mathbf{B}$, 
-		we can take the isosurfaces of $\chi$. 	
-	\end{alertblock}
-
-\end{frame}
-	
-\begin{frame}{Angle between field from VMEC and Dommaschk potential}
-	\begin{itemize}
-		\item As a test, Florian Hindenlang took Dommaschk potential fitted to $\mathbf{B}$-field from VMEC for W7A 
-			with zero toroidal current and zero pressure
-			\footnote{Data provided by Nikita Nikulsin and Rohan Ramasamy, see Nikulsin et al (2022) - 
-			"JOREK3D: An extension of the JOREK nonlinear MHD code 
-			to stellarators"}.
-		\item Angle between $\mathbf{B}$ and $\nabla \chi$ less than $10^{-3}$
-	\end{itemize}
-
-	\begin{alertblock}{}
-		The equilibrium field $\mathbf{B}$ is almost orthogonal to the isosurfaces of $\chi$
+		\begin{itemize}
+			\item Already implemented in Psydac and successfully applied to several problems. 
+			\item Probably easier to generalize to complicated geometries than other approaches
+					like using different splines e.g.
+		\end{itemize}
 	\end{alertblock}
-
-	% The following plots are the angle 
-	% in degrees between 
-	% the magnetic field of W7A with zero toroidal current and zero pressure computed using 
-	% VMEC and the field 
-	% calculated from the Dommaschk potential 
-	% \footnote{Coefficients provided by Nikita Nikulsin and Rohan Ramasamy, see Nikulsin et al (2022) - 
-	% "JOREK3D: An extension of the JOREK nonlinear MHD code 
-	% to stellarators"}.
-	% Since the angle is almost zero, the magnetic field
-	% is almost orthogonal on the isosurfaces.
-
-
-	% \begin{columns}[
-	% 	onlytextwidth,%otherwise the whole pagewidth would be used
-	% 	c, %align vertically centered
-	% 	]
-	% \begin{column}{.5\linewidth}
-	% 		\includegraphics[width=\textwidth]{plots/Bfield_dommaschk_vs_gvec1.png}
-	% \end{column}
-	% \begin{column}{.5\linewidth}
-	% 		\includegraphics[width=\textwidth]{plots/Bfield_dommaschk_vs_gvec2.png}
-	% \end{column}
-
-\end{frame}
-
-\begin{frame}{Plot of angle between $\mathbf{B}$ and $\nabla \chi$}
-	Plots show the angle between $\mathbf{B}$ and $\nabla \chi$ in degrees on two poloidal cuts
-	\begin{figure}
-		\centering
-		\includegraphics[width=0.72\textwidth]{plots/Bfield_dommaschk_vs_gvec_2subplots.png}
-	\end{figure}
-
 \end{frame}
 
-\begin{frame}{Comparison with taking only $\varphi$ direction}
+\begin{frame}{Patch data -- Exchange}
 	\begin{itemize}
-		\item Plots show the angle between $\mathbf{e}_\varphi$ and $\mathbf{B}$ on the same 
-				poloidal cuts
-		\item The angle is larger by 4 orders of magnitude in our example
+		\item Mesh-points,
+		\item Local sums to compute the derivatives at the interfaces thanks to this 
+			type of sum of spline values (Advection) ({\color{red} What is $\alpha$, please check!})
+				\begin{align*}
+					\sum_{x_i \in \text{global space}} \alpha_i s(x_i) = \sum_{p \in\text{Patches}} 
+							\sum_{x_i \in p} \alpha_i s(x_i),
```

Comment:
```suggestion
					\sum_{x_i \in \text{global space}} \alpha_i(\{x_j\}_j) s(x_i) = \sum_{p \in\text{Patches}} 
							\sum_{x_i \in p} \alpha_i(\{x_j\}_j)  s(x_i),
```

Reply:
I dont know how to make a suggestion on several code lines. But we can write like this. \alpha are just doubles which depend on the mesh points.